Content.Server/GameObjects/Components/Projectiles/HitscanComponent.cs

```diff
@@ -94,7 +95,7 @@ public void FireEffects(IEntity user, float distance, Angle angle, IEntity hitEn
                 EntitySystem.Get<AudioSystem>().PlayAtCoords(_soundHitWall, user.Transform.Coordinates.Offset(offset));
             }
 
-            Timer.Spawn((int) _deathTime.TotalMilliseconds, () =>
+            Owner.SpawnTimer((int) _deathTime.TotalMilliseconds, () =>
             {
                 if (!Owner.Deleted)
                 {
```

Comment:
Timer callback doesn't execute if owner's deleted now right? So I think this can be deleted?

Reply:
It wouldn't, but where else would the projectile get deleted if not here?

Reply:
I think explosions can delete it?

At any rate I made hitscans not-entities on my gun prediction branch so this is likely to get bulldozed at some point.